src/modules/missions/checkers/linkChecker.js

```diff
@@ -0,0 +1 @@
+module.exports = ({ missionId }, options) => options.missionId === missionId;
```

Comment:
... should has some db call specific code? or something that need for link checking